src/index.ts

```diff
@@ -133,13 +145,14 @@ function path(template: string, params: ParamMap) {
 }
 
 /**
- * Joins two strings using a separator. If the separator occurs at the concatenation
- * boundary in either of the strings, it is removed. This prevents accidental duplication
- * of the separator.
+ * Joins two strings using a separator.
+ * If the separator occurs at the concatenation boundary in either parameter, it is removed.
```

Comment:
"in either parameter" is incorrect; there are three parameters. I would keep my original version here.